### PR TITLE
feat(controller): allow admins to change a user's password.

### PR DESCRIPTION
**Pull request by Joshua-Anderson**
Now they no longer need to know a user's password.

### CODE REVIEW

**Comment by bacongobbler** [comment]
this will probably merit an API bump as well as doc changes. Have we bumped the API this release? I don't think so. Regardless, the implementation looks good :)

**Comment by carmstrong** [comment]
If we're bumping the API version this release, we should merge #3999 as well.

**Comment by Joshua-Anderson** [comment]
@bacongobbler Bumped the api to v1.6.

File: docs/reference/index.rst [hunk 1]
```diff
@@ -20,3 +20,4 @@ Reference Guide
     api-v1.3
     api-v1.4
     api-v1.5
+    api-v1.5
```
Comment (carmstrong) [inline docs/reference/index.rst]: Copypaste error

**Comment by carmstrong** [comment]
#3851 is related

**Comment by carmstrong** [comment]
also related: #3976, if you're looking for work, @Joshua-Anderson ;)

**Comment by Joshua-Anderson** [comment]
@carmstrong Thanks for the catch. Done!

**Comment by krancour** [comment]
Code LGTM

**Comment by bacongobbler** [comment]
code LGTM.

**Pull request merged**